install/share/freeipa-server.template

```diff
@@ -0,0 +1,9 @@
+[plugins]
```

Comment:
The build system doesn't pick up new files automatically. You have to add this file to ``Makefile.am`` and maybe ``freeipa.spec.in``